velox/dwio/parquet/tests/ParquetReaderTestBase.h

```diff
@@ -39,7 +39,7 @@ class ParquetReaderTestBase : public testing::Test {
   }
 
   static RowTypePtr dateSchema() {
-    return ROW({"date"}, {DATE()});
+    return ROW({"int"}, {INTEGER()});
```

Comment:
Why is this change needed  ?

 static RowTypePtr dateSchema() {
    return ROW({"date"}, {DATE()}); seems reasonable.